recipes/gc/all/patches/update-cmake.patch

```diff
@@ -0,0 +1,834 @@
+--- CMakeLists.txt
```

Comment:
This is a complete new CMake file. Did you implement it, or did you find it in some repository?

Reply:
This is the cmake script form current master.
I could not let autotools create a shared Visual Studio build.

Reply:
I see, so it's totally valid. thanks.

Reply:
Thanks for reviewing btw.